jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/DockerClientParameters.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2018 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib.gradle;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Map;
+import javax.annotation.Nullable;
+import org.gradle.api.tasks.Input;
+import org.gradle.api.tasks.Optional;
+
+/**
+ * Object in {@link BuildDockerTask} that configures the Docker executable and the additional
+ * environment variables to use when executing the executable.
+ */
+class DockerClientParameters {
```

Comment:
I believe this needs to be public for the `build.gradle` to use it correctly.

Reply:
It worked with `build.gradle`, but I'll change it to public anyway.